app/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:1.18
-WORKDIR /go/src/nearform/3-tiers-cloudrun
+FROM golang:1.16
```

Comment:
You can undo this changes as we used a different folder to store the cloudfunction version of app